tests/unit/modules/test_file.py

```diff
@@ -338,29 +342,53 @@ def test_uncomment(self):
 
 
 class FileBlockReplaceTestCase(TestCase, LoaderModuleMockMixin):
+    @classmethod
+    def setUpClass(self):
+        if salt.utils.platform.is_windows():
+            self.original_check_perms = filemod.check_perms
+            filemod.check_perms = win_file.check_perms
+
+    @classmethod
+    def tearDownClass(self):
+        if salt.utils.platform.is_windows():
+            filemod.check_perms = self.original_check_perms
```

Comment:
Does overriding filemod.check_perms then restoring it from the class actually work?

Reply:
Who knows...